ansible/files/adminapi.sudoers.conf

```diff
@@ -25,3 +26,4 @@ Cmnd_Alias PGBOUNCER = /bin/systemctl start pgbouncer.service, /bin/systemctl st
 %adminapi ALL= NOPASSWD: POSTGREST
 %adminapi ALL= NOPASSWD: GOTRUE
 %adminapi ALL= NOPASSWD: PGBOUNCER
+%adminapi ALL= NOPASSWD: PG_EGRESS_COLLECT
```

Comment:
changes in this file are not needed afaict?

Reply:
ok, removed.